.circleci/config.yml

```diff
@@ -63,33 +63,13 @@ jobs:
       - restore_cache:
           key: v7-dependency-cache-{{ checksum "yarn.lock" }}
       - run: yarn dist:libs
-      - persist_to_workspace:
-          root: '.'
-          paths:
-            - packages/core/dist
```

Comment:
notably, I forgot to persist `packages/core/lib` which includes `variables.less` and `variables.scss`